### PR TITLE
Geneva trace/log exporter include wrappers

### PR DESCRIPTION
**Pull request by lalitb**
This PR adds the wrapper/convenience include files over [ETW ](https://github.com/open-telemetry/opentelemetry-cpp/tree/main/exporters/etw) exporter, and NuGet package generation script.

### CODE REVIEW

File: exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * - KeywordName
+ * - TaskName
+ * - ChannelName
+ * - EventMessage
+ * - ActivityId
+ * - Pid
+ * - Tid
+ *
+ */
+#  define ETW_FIELD_VERSION           "env_ver"               /* Event version           */
```
Comment (ThomsonTan) [inline exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h:36]: Are these definition different than the one defined in ETW exporter?

File: exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+#  define ETW_FIELD_TYPE              "env_type"              /* Event type              */
+#  define ETW_FIELD_NAME              "env_name"              /* Event name              */
+
+#ifndef HAVE_FIELD_TIME
+#define HAVE_FIELD_TIME
+#endif
+#  define ETW_FIELD_TIME              "env_time"              /* Event time at envelope  */
```
Comment (ThomsonTan) [inline exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h:43]: Should this `ETW_FIELD_TIME` be included in the above `#ifndef HAVE_FIELD_TIME / #endif`?

File: exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h [hunk 3]
```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 (c) Microsoft Corporation.  All rights reserved.
```
Comment (ThomsonTan) [inline exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h:1]: Update year of copyright?

File: exporters/geneva-trace/tools/build-nuget.cmd [hunk 4]
```diff
@@ -0,0 +1,20 @@
+@echo off
+pushd "%~dp0"
+set "PATH=%CD%;%PATH%"
+if not exist "..\packages" mkdir "..\packages"
+if not defined PackageVersion (
+  echo "Set the PackageVersion before proceeding."
```
Comment (ThomsonTan) [inline exporters/geneva-trace/tools/build-nuget.cmd:6]: nit: check version before creating "..\packages"?

File: exporters/geneva-trace/tools/build-nuget.cmd [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  exit
+)
+
+REM This script assumes that the main OpenTelemetry C++ repo
```
Comment (ThomsonTan) [inline exporters/geneva-trace/tools/build-nuget.cmd:10]: Is contrib repo 2 levels up than the main repo?

File: exporters/geneva-trace/tools/nuget/OpenTelemetry.Cpp.Geneva.nuspec [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  <metadata>
+    <id>OpenTelemetry.Cpp.Geneva</id>
+    <version>0.0.0</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
+    <icon>opentelemetry-icon-color.png</icon>
+    <projectUrl>https://opentelemetry.io/</projectUrl>
+    <description>OpenTelemetry C++ SDK with Geneva Exporter</description>
```
Comment (ThomsonTan) [inline exporters/geneva-trace/tools/nuget/OpenTelemetry.Cpp.Geneva.nuspec:11]: Should "SDK" be "API/SDK"?

File: exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h (hunk 3 above)
Comment (lalitb) [reply exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h]: Good point, have updated for all the files now. And Copyright is with OpenTelemetry, just like the dotnet exporter (https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/6d4fb62ee904582f2b0787ee5e02d227558c5359/src/OpenTelemetry.Exporter.Geneva/AssemblyInfo.cs#L2)

File: exporters/geneva-trace/tools/nuget/OpenTelemetry.Cpp.Geneva.nuspec (hunk 6 above)
Comment (lalitb) [reply exporters/geneva-trace/tools/nuget/OpenTelemetry.Cpp.Geneva.nuspec]: Yeah made it API/SDK now.

**Review by ThomsonTan — APPROVED** [review]

**Pull request merged**